auslib/admin/views/base.py

```diff
@@ -51,6 +51,10 @@ def decorated(*args, **kwargs):
                 msg = "Permission denied to perform the request. {}".format(e.message)
                 logging.warning(msg)
                 return Response(status=403, response=json.dumps({"exception": msg}), mimetype="application/json")
+            except ValueError as e:
+                msg = "Error. {}".format(e.message)
+                logging.warning(msg)
+                return Response(status=403, response=json.dumps({"exception": msg}), mimetype="application/json")
```

Comment:
This should be a 400 actually - 403 is specifically for cases where somebody is not authorized to do something (eg, because of permissions).